### ISSUE TITLE
Server can not start when ledger is installed

### ISSUE THREAD

**Issue by DJMalachite** [thecolonel63/server-side-replay-recorder#29]
Hello there, I was hoping to use this mod for my SMP but once the server will not start it just throws an error.

I have had the server start with the replay mod and every other mod I have except ledger 

Are they straight incompatible or is there something that can be done to fix this?

here is the logs

```[00:41:44] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'serversidereplayrecorder'!
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.14.21.jar:?]
        at net.minecraft.server.Main.main(Main.java:112) ~[server-intermediary.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.14.21.jar:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.14.21.jar:?]
Caused by: java.lang.RuntimeException: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.jsr310.JavaTimeModule not found
        at com.thecolonel63.serversidereplayrecorder.ServerSideReplayRecorderServer.loadConfig(ServerSideReplayRecorderServer.java:59) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at com.thecolonel63.serversidereplayrecorder.ServerSideReplayRecorderServer.onInitialize(ServerSideReplayRecorderServer.java:129) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.21.jar:?]
        ... 7 more
Caused by: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.jsr310.JavaTimeModule not found
        at java.util.ServiceLoader.fail(ServiceLoader.java:593) ~[?:?]
        at java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1219) ~[?:?]
        at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
        at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
        at java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
        at java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
        at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1131) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1115) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:1165) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at com.thecolonel63.serversidereplayrecorder.ServerSideReplayRecorderServer.loadConfig(ServerSideReplayRecorderServer.java:52) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at com.thecolonel63.serversidereplayrecorder.ServerSideReplayRecorderServer.onInitialize(ServerSideReplayRecorderServer.java:129) ~[serversidereplayrecorder-1.1.5-mc1.20.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.21.jar:?]
        ... 7 more```

**Comment by thecolonel63** [comment]
I saw another issue similar to this one - I'm gonna keep this open until I can at least figure out what's causing the incompatibility.

**Comment by DJMalachite** [comment]
> I saw another issue similar to this one - I'm gonna keep this open until I can at least figure out what's causing the incompatibility.

Awesome keep me posted 👍

**Comment by thecolonel63** [comment]
Turns out the guy that rewrote the config logic did it in such a way that the library used caused issues when other mods also used it - probably something that can be fixed in a couple days or so. When this is fixed, I'll publish an updated mod version and close this issue.

**Comment by Cyclopropinon** [comment]
for both 1.19.4 and 1.20.1 pls :)

**Comment by thecolonel63** [comment]
As of release version 1.1.6, this issue has been fixed.

Backporting the fix to 1.19.4 will occur when a better system of supporting multiple versions at the same time is found to be effective.

**Comment by DJMalachite** [comment]
Amazing thank you @thecolonel63
